infrastructure/sit_loader.py

```diff
@@ -11,18 +11,17 @@
 
 class SITLoader(object):
 
-    TROPOSPHERE_CONFIGS = SITHelper().get_configs('troposphere')
-    STACK_NAME = TROPOSPHERE_CONFIGS['stack_name']
-    TAG_VALUE = TROPOSPHERE_CONFIGS['tag_value']
-    AMI_ID = TROPOSPHERE_CONFIGS['ami_id']
-    SUBNET = TROPOSPHERE_CONFIGS['subnet']
-    SECURITY_GROUPS = TROPOSPHERE_CONFIGS['security_groups']
-    KEY_NAME = TROPOSPHERE_CONFIGS['key_name']
-    AMI_URL = TROPOSPHERE_CONFIGS['ami_url']
-
-    def __init__(self):
-        self.cf_helper = CFHelper()
-        self.sit_template = SITTemplate()
+    def __init__(self, session=None, configs_directory='configs'):
```

Comment:
switch the order of the variables to be consistent (all other classes have session after configs_directory)